pkg/okteto/pipeline.go

```diff
@@ -457,62 +450,3 @@ func (c *pipelineClient) GetResourcesStatus(ctx context.Context, name string) (m
 func getResourceFullName(kind, name string) string {
 	return strings.ToLower(fmt.Sprintf("%s/%s", kind, name))
 }
-
-// StreamLogs retrieves logs from the pipeline provided and prints them, returns error
-func (c *pipelineClient) StreamLogs(ctx context.Context, name, actionName string) error {
```

Comment:
Be aware of the change @jLopezbarb is doing related to include `namespace` as an option in the dependencies field of the manifest

Reply:
rebased!